.env.example

```
@@ -3,11 +3,27 @@
-STORAGE_DIR=./uploads
 STATIC_DIR=./static
 DOMAIN=
 USE_HTTPS=false
 ADMIN_PASSWORD=
 MCP_MAX_BODY=35mb
 MCP_MAX_UPLOAD_BYTES=26214400
 MCP_ALLOWED_ORIGINS=
+
+# Where uploads are kept: local | s3. Defaults to s3 once S3_BUCKET is set.
+STORAGE_DRIVER=local
+STORAGE_DIR=./uploads
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The STORAGE_DIR key should go before the STORAGE_DRIVER key

(UnorderedKey)